apps/perf-test-react-components/package.json

```diff
@@ -19,14 +19,14 @@
   },
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28426&commitId=415002306ee8d8a2bc49f6d020dbaedb990870a2&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popup.Popup Example Offset.Opens a popup.chromium.png|1252| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>